src/osd/PrimaryLogPG.cc

```diff
@@ -4748,8 +4762,11 @@ int PrimaryLogPG::do_extent_cmp(OpContext *ctx, OSDOp& osd_op)
   read_op.op.extent.truncate_size = op.extent.truncate_size;
 
   int result = do_osd_ops(ctx, read_ops);
-  if (result < 0) {
-    derr << "do_extent_cmp do_osd_ops failed " << result << dendl;
+  if (result == -ENOENT) {
```

Comment:
This is the sync read path -- ```do_osp_ops``` would invokes and returns the result from ```do_read```, which returns the result from ```pgbackend->objects_read_sync```